packages/app/src/overview/components/overview-screens/pure-overview/overview-content.component.tsx

```diff
@@ -22,11 +22,6 @@ interface OverviewContentProps {
   readonly memorySpace: MemorySpace
   readonly networkName: string
   readonly networkLevel: number | undefined
-  readonly pureOsAvailable: boolean
-  readonly pureOsDownloaded: boolean
-  readonly checkForUpdatePerformed: boolean
-  readonly checkForUpdateInProgress: boolean
-  readonly checkForUpdateFailed: boolean
   readonly osVersion: string
   readonly caseColour: CaseColor | undefined
   readonly lastBackupDate: Date | undefined
```

Comment:
I know this is not your code, but why do we need those readonly here and we don't use those anywhere else? :D Maybe we could throw them out? What do you think?

Reply:
I think we can get rid of those :D